wrapper/src/main/java/software/amazon/jdbc/util/RdsUtils.java

```diff
@@ -284,7 +284,7 @@ public String getRdsClusterHostUrl(final String host) {
     }
     final Matcher limitlessMatcher = AURORA_LIMITLESS_CLUSTER_PATTERN.matcher(host);
```

Comment:
I'm not sure we need to include limitless cluster (that has quite unique feature) here. Let's discuss.